.github/workflows/test-agent.yml

```
@@ -0,0 +1,143 @@
+# Copyright 2020 New Relic Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: test-pull-request
+
+#
+# Controls when the action will run. 
+#
+on:
+  #
+  # Triggers the workflow on push or pull request events but only for the main branch
+  #
+  push:
+    branches: 
+      - main
+      - 'dev'
+  pull_request:
+    branches: 
+      - main
+      - 'dev'
+
+jobs:
+  daemon-unit-tests:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: newrelic/nr-php-agent-builder
+      IMAGE_TAG: make-go
+      IMAGE_VERSION: v1
+    strategy:
+      matrix:
+        platform: [gnu, musl]
+        arch: [amd64, arm64]
+    steps:
+      - name: Checkout newrelic-php-agent code
+        uses: actions/checkout@v3
+        with:
+          path: php-agent
+      - name: Enable arm64 emulation
+        if: ${{ matrix.arch == 'arm64' }}
+        run: |
+          docker run --privileged --rm tonistiigi/binfmt --install arm64
```

since this is only one line, you should be able to omit the literal block scalar
```suggestion
        run: docker run --privileged --rm tonistiigi/binfmt --install arm64
```

---

Actually, looking at this again, I think I prefer the "cleaner" look of the command being on it's own line